Readme.md

```diff
@@ -1 +1,79 @@
-HMC emPress codebase for Spring 2020 Comp Bio research
+# HMC eMPRess Codebase for Spring 2020 Research
+
+## Overview
+eMPRess (in Python 3.7.4) supports the following functionality:
+* DTL Reconciliation - `reconcile`
+* Costscape - `costscape`
+* Pair-distance Histogram - `histogram`
+* Cluster MPR - `clumpr`
```

Comment:
We should add an instruction on how to install dependencies. I will open a new issue for this. #5